website/src/pages/showcase.js

```diff
@@ -99,11 +99,12 @@ const Showcase = () => {
         </a>
         <p>
           A curated list of{' '}
-          <a href="https://github.com/ReactNativeNews/React-Native-Apps">
+          <a
+            key="demo-apps"
```

Comment:
doesn't seem to work

try wrapping this link in a `<span>` instead 🤪 generally changing slightly the markup can solve these weird hydration issues

Reply:
thank you so much  :) !.